src/myRandom.cpp

```diff
@@ -3,18 +3,18 @@
 #include <string>
 #include <random>
 
+static std::random_device rd;  // Seed source
+static std::mt19937 gen(rd());  // Mersenne Twister RNG
+
 int myRandom::generateRandomInt(int min, int max) {
-    std::random_device rd;  // Seed source
-    std::mt19937 gen(rd()); // Mersenne Twister RNG
+    // This one can't be static because it depends on the inputs to the function.
     std::uniform_int_distribution<> distrib(min, max);
     return distrib(gen);
 }
 
 std::string myRandom::generateRandomString(size_t length) {
     const std::string chars = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789";
-    std::random_device rd;  // Seed source
-    std::mt19937 gen(rd()); // Mersenne Twister RNG
-    std::uniform_int_distribution<> distrib(0, chars.size() - 1);
+    static std::uniform_int_distribution<> distrib(0, chars.size() - 1);
```

Comment:
function-level `static` means to create this instance once and then reuse it on every call to the function. The object will maintain state. That's good for random number generators if you value reproducibility with a user-controlled random seed (for reproducing test results).

More generally, it's good to avoid unnecessary allocations when some object is known to be "the same" each time through the function.